backend/btrixcloud/operator.py

```diff
@@ -495,7 +495,7 @@ async def mark_finished(
         if stats:
             kwargs["stats"] = stats
 
-        await update_crawl(self.crawls, crawl_id, **kwargs)
+        crawl = await update_crawl(self.crawls, crawl_id, **kwargs)
```

Comment:
The crawl here is a different type of object, CrawlSpec, not our usual Crawl.
(Maybe should use different variable for this to avoid confusion, sorry)

Reply:
No worries! Reverted